api/routes/queues/routes.py

```diff
@@ -16,24 +20,44 @@ def post_submitted_application_to_assessment(self, application_id=None):
         if application_with_form_json["status"] == "SUBMITTED":
             application_attributes = {
                 "application_id": {"StringValue": application_id, "DataType": "String"},
+                "S3Key": {
+                    "StringValue": ASSESSMENT_S3_KEY_CONST,
+                    "DataType": "String",
+                },
             }
 
-            # Submit message to queue, in a future state this can
-            # trigger the assessment service to import the application
-            #  (currently assessment is using a CRON timer to pick up messages,
-            # not a webhook for triggers)
-
-            message_submitted_id = _SQS_CLIENT.submit_single_message(
-                queue_url=Config.AWS_SQS_IMPORT_APP_PRIMARY_QUEUE_URL,
-                message=application_with_form_json,
-                extra_attributes=application_attributes,
-                message_group_id="import_applications_group",
-                message_deduplication_id=str(uuid4()),  # ensures message uniqueness
-            )
-
-            return f"Message queued, message_id is: {message_submitted_id}.", 201
+            """
```

Comment:
This code seems to do the same thing as https://github.com/communitiesuk/funding-service-design-application-store/pull/343/files#diff-92de12e32db2cfd608043c63a25bbd576f0cafb49cb55d8d46f747270936a080R252 above, is that right?